airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -59,10 +60,19 @@ def get_dag_details(dag_id):
 
 @security.requires_access([(permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG)])
 @format_parameters({'limit': check_limit})
-def get_dags(limit, offset=0):
+def get_dags(limit, sort, offset=0, order_by='dag_id'):
     """Get all DAGs."""
+    columns = [i.name for i in DagModel.__table__.columns]  # pylint: disable=no-member
+    if order_by not in columns:
```

Comment:
I think this will work

```suggestion
    if order_by not in (i.name for i in DagModel.__table__.columns):  # pylint: disable=no-member
```

Reply:
Also slightly more efficient (only need to go through the columns once).